docs/getting-started.md

```diff
@@ -24,40 +24,47 @@ You can optionally persist the output of the `onit completion` command to your s
 
 ## Usage
 
+### Development Cluster
+
+To setup a cluster for development, after having pushed all the new images to `kind` run:
+```bash
+onit create cluster --set onos-cli.enabled=true
+```
+
 ### Integration Tests
 
 To run a suite of tests, use the `onit run test` command, providing a test image to run:
 
 ```bash
-onit test --image onosproject/onos-tests:latest
+onit test --image <test-image>
```

Comment:
are there many `test-image`s around? how do I know what they are?

Reply:
Yes they are. Each subsystem has its own test image and the users also can built one. It is already explained in the docs but I will provide a pointer here to make it clear.